### PR TITLE
[Fix] Fix add_graph in pavi

### PR DESCRIPTION
**Pull request by gengenkai**

### CODE REVIEW

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
LGTM, can be merged if the PR is validated by downstream requests.

File: mmcv/runner/hooks/logger/pavi.py [hunk 1]
```diff
@@ -95,3 +95,15 @@ def after_run(self, runner):
                     tag=self.run_name,
                     snapshot_file_path=ckpt_path,
                     iteration=iteration)
+
+    @master_only
```
Comment (ZwwWayne) [inline mmcv/runner/hooks/logger/pavi.py:99]: add `@torch.no_grad` above this line

File: mmcv/version.py [hunk 2]
```diff
@@ -1,29 +0,0 @@
-# Copyright (c) Open-MMLab. All rights reserved.
```
Comment (ZwwWayne) [inline mmcv/version.py:1]: Please recover this file

**Comment by OpenMMLab-gd** [comment]
Dear gengenkai,
First of all, we want to express our gratitude for your significant PR in the MMCV project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.
If you are Chinese or have WeChat，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. We look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG
Thank you again for your contribution❤
Best regards！ @gengenkai

**Pull request merged**